src/main/features/response/draft/responseDraft.ts

```diff
@@ -29,6 +31,7 @@ export class ResponseDraft extends Draft implements Serializable<ResponseDraft>
       this.counterClaim = CounterClaim.fromObject(input.counterClaim)
       this.defendantDetails = new Defendant().deserialize(input.defendantDetails)
       this.lastUpdateTimestamp = input.lastUpdateTimestamp
+      this.howMuchOwed = HowMuchOwed.fromObject(input.howMuchOwed)
```

Comment:
The pattern we follow is to use `fromObject` for form values conversion and `deserialize` for deserializing values retrieved from API. Please update this to use `deserialize`.